### PR TITLE
fix clippy error: cast to more strict alignment, warning: not consumed copy

### PR DESCRIPTION
**Pull request by jackcmay**
Interested in using rBPF for a project I'm working on so fixed a couple of clippy errors/warnings.  If this project is still active and these changes are approved would like to fix more

- pulled byteorder dependency back in to avoid unsafe code and to fix clippy casting errors
- removed copy of operand that was not consumed

### CODE REVIEW

**Comment by badboy** [comment]
/o\ I did? Will review it now. Sorry for the delay.

File: src/ebpf.rs [hunk 1]
```diff
@@ -18,6 +18,9 @@
 //! <https://www.kernel.org/doc/Documentation/networking/filter.txt>, or for a shorter version of
 //! the list of the operation codes: <https://github.com/iovisor/bpf-docs/blob/master/eBPF.md>
 
+extern crate byteorder;
```
Comment (badboy) [inline src/ebpf.rs]: This should not be needed, it's imported on the root level already.

File: src/ebpf.rs [hunk 2]
```diff
@@ -18,6 +18,9 @@
...
+use self::byteorder::{ByteOrder, LittleEndian};
```
Comment (badboy) [inline src/ebpf.rs]: You can drop the `self::` here then.

File: src/jit.rs [hunk 3]
```diff
@@ -297,7 +297,7 @@ fn emit_load_imm(jit: &mut JitMemory, dst: u8, imm: i64) {
 
 // Store register src to [dst + offset]
 #[inline]
-fn emit_store(jit: &mut JitMemory, size: OperandSize, src: u8, dst: u8, offset: i32) {
```
Comment (badboy) [inline src/jit.rs]: We should simply make `OperandSize` copy in `src/jit.rs`

File: src/lib.rs [hunk 4]
```diff
@@ -22,6 +22,9 @@
 use std::u32;
 use std::collections::HashMap;
 
+extern crate byteorder;
+use self::byteorder::{ByteOrder, LittleEndian};
```
Comment (badboy) [inline src/lib.rs]: Keep the `extern crates` together and move the `use byteorder...` line below that then. The `self::` should not be necessary anymore.

Comment (jackcmay) [reply src/lib.rs]: Would you prefer to keep all the `use`'s together and move the two existing`use std::` lines below the group of `extern`'s

Comment (badboy) [reply src/lib.rs]: That's my preferable style, yes.

1. extern
2. use std
3. use external stuff

**Pull request merged**